gum-gum-htb.js

```diff
@@ -209,8 +209,9 @@ function GumGumHtb(configs) {
 
         /* ------- Put GDPR consent code here if you are implementing GDPR ---------- */
         var privacyEnabled = ComplianceService.isPrivacyEnabled();
-        queryObj.gdprApplies = privacyEnabled
-        if (privacyEnabled) {
+        var gdprConsent = ComplianceService.gdpr.getConsent();
+        queryObj.gdprApplies = gdprConsent.applies
```

Comment:
Hi,
I would recommend to put this line inside the `if (privacyEnabled && gdpr.applies)` scope, but if this is your use case then that's fine as well.